kong/basic_auth_service.go

```diff
@@ -38,7 +52,39 @@ func (s *BasicAuthService) Create(ctx context.Context,
 	consumerUsernameOrID *string, basicAuth *BasicAuth,
 ) (*BasicAuth, error) {
 	cred, err := s.client.credentials.Create(ctx, "basic-auth",
-		consumerUsernameOrID, basicAuth)
+		consumerUsernameOrID, basicAuth, false)
```

Comment:
Could we call `CreateWithOptions` inside `Create` to make code simpler?

Reply:
Ohh. I understood your point now. I will update the code.